package.json

```diff
@@ -108,6 +124,18 @@
 			{
 				"command": "file-switch.relatedFiles",
 				"title": "File Switch: Related Files"
+			},
+			{
+				"command": "path-copy.copyContextRelative",
+				"title": "File Ops: Copy Path"
+			},
+			{
+				"command": "path-copy.pasteContextRelative",
+				"title": "File Ops: Paste Relative Path"
+			},
+			{
+				"command": "path-copy.copyExploreRelative",
```

Comment:
Why 2 separate methods for copying. The same fn can be reused.
Copy command will come before paste
Remove `relative` from the function name too